pkg/cli/start.go

```diff
@@ -1045,6 +1045,9 @@ func doShutdown(ctx context.Context, c serverpb.AdminClient, onModes []int32) er
 	// out, or perhaps drops the connection while waiting). To that end, we first
 	// run a noop DrainRequest. If that fails, we give up.
 	if err := checkNodeRunning(ctx, c); err != nil {
+		if grpcutil.IsClosedConnection(err) {
```

Comment:
As @bdarnell asked on https://github.com/cockroachdb/cockroach/pull/26163, does this swallow the error if you run `quit` against a server that isn't running?

Reply:
answered to ben there. if the server does not exist the thing still fails.